packages/devextreme/scss/widgets/base/textArea/_index.scss

```diff
@@ -1,8 +1,6 @@
 // adduse
-
```

Comment:
This too.

Reply:
this change sir is going to cause some lint errors.

Reply:
Correction: [Commit](https://github.com/DevExpress/DevExtreme/pull/26885/commits/8c3cd0d674579d15406941118b5f92d97ddecb40)